Tests/WinAppDriverTests/AppDriverOptionsTest.swift

```diff
@@ -0,0 +1,34 @@
+import TestsCommon
+import WinSDK
+import XCTest
+
+@testable import WebDriver
+@testable import WinAppDriver
+
+class AppDriverOptionsTest: XCTestCase {
+    func tempFileName() -> String {
+        return FileManager.default.temporaryDirectory.appendingPathComponent(UUID().uuidString)
+            .appendingPathExtension("txt").path
+    }
+
+    /// Tests that redirecting stdout to a file works.
+    func testStdoutRedirectToFile() throws {
+        let outputFile = try {
+            // Start a new instance of msinfo32 and write the output to a file.
+            let outputFile = tempFileName()
+            let _ = try MSInfo32App(
+                winAppDriver: WinAppDriver.start(
+                    outputFile: outputFile
+                ))
+            return outputFile
+        }()
+
+        // Read the output file.
+        let output = try String(contentsOfFile: outputFile, encoding: .utf16LittleEndian)
```

Comment:
Does WinAppDriver write as utf-16 when redirected to a file? That's unusual

Reply:
It writes UTF-16 either way. It's usually not noticeable because Windows Terminal handles either.

Reply:
Terminals are supposed to decode stdout according to the `GetConsoleOutputCP()` of the console, so outputting UTF-16 would result in garbled contents. 🤔🤔🤔

Reply:
Probably witchcraft involved:
```powershell
PS > $output = & 'C:\Program Files (x86)\Windows Application Driver\WinAppDriver.exe'

PS > [system.Text.Encoding]::ASCII.GetBytes($output)
87
0
105
0
[...]

PS > $output = echo potato
PS > [system.Text.Encoding]::ASCII.GetBytes($output)
112
111
116
97
116
111
```

Reply:
I don't know if that's a valid test but it is puzzling!